baldrick/github/github_auth.py

```diff
@@ -112,7 +112,7 @@ def github_request_headers(installation):
     return headers
 
 
-def repo_to_installationid_mapping():
+def repo_to_installation_id_mapping():
```

Comment:
Just a general point, this function is *very* expensive in time and API calls to GitHub we should think about trying to cache it. However, we need to be careful about how we invalidate that cache.